.github/workflows/release-sanity-check.yml

```
@@ -0,0 +1,61 @@
+
+      - name: Install from built wheel in clean venv
+        run: |
+          python -m venv /tmp/sanity-venv
+          /tmp/sanity-venv/bin/pip install dist/*.whl
+
+      - name: Verify import works
+        run: |
+          /tmp/sanity-venv/bin/python -c "
+          import qdrant_client
+          from importlib.metadata import version
+print(f'qdrant-client {version("qdrant-client")} imported successfully')
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)